docs/configuration.md

```diff
@@ -3029,7 +3029,10 @@ can be found on the pages for each mode:
 Certain Spark settings can be configured through environment variables, which are read from the
 `conf/spark-env.sh` script in the directory where Spark is installed (or `conf/spark-env.cmd` on
 Windows). In Standalone and Mesos modes, this file can give machine specific information such as
-hostnames. It is also sourced when running local Spark applications or submission scripts.
+hostnames. It is also sourced when running local Spark applications or submission scripts. For
+pyspark applications, the environment variable `_PYSPARK_DRIVER_SYS_EXECUTABLE` will be set to
+the python interpreter's `sys.executable`, which will allow further customization based on the
+user's virtual environment.
```

Comment:
nit: Can we use official naming for Python and PySpark ??

- pyspark -> PySpark
- python -> Python

Reply:
done @ e747a2f

Reply:
done!